classes/Customer.php

```diff
@@ -265,6 +269,33 @@ public function add($autoDate = true, $nullValues = true)
         return $success;
     }
 
+    /**
+     * Adds current Customer as a new Object to the database.
+     *
+     * @param bool $autoDate Automatically set `date_upd` and `date_add` columns
+     * @param bool $nullValues Whether we want to use NULL values instead of empty quotes values
+     *
+     * @return bool Indicates whether the Customer has been successfully added
+     *
+     * @throws PrestaShopDatabaseException
+     * @throws PrestaShopException
+     */
+    public function addWs($autodate = true, $null_values = false)
+    {
+        if (CustomerCore::customerExists($this->email)) {
+            WebserviceRequest::getInstance()->setError(
+                500,
+                $this->trans('The email is already used, please choose another one',
+                array(), 'Admin.Notifications.Error'),
+                140
```

Comment:
How did you come up with this number?

Reply:
I was looking for some info for the new issue of this error, I asked @matks  if there is any directive, the answer was no. So we agreed on a number that still does not exist .... and may change in the future

Reply:
Yep that sounds right to me. @eternoendless is that OK for you ?